lib/manager/nuget/extract.ts

```diff
@@ -74,7 +74,7 @@ export async function extractPackageFile(
 ): Promise<PackageFile | null> {
   logger.trace({ packageFile }, 'nuget.extractPackageFile()');
 
-  const { localDir } = GlobalConfig.get();
+  const localDir = GlobalConfig.get('localDir');
```

Comment:
Why do we need local dir? 🤔 Looks like it's not used in `getConfiguredRegistries​`

Reply:
Oh, it became reduntat. I'll remove it, thank you.